src/components/Navigation.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navigation = () => {
+  return (
+    <nav>
+      <NavLink to="/list" activeClassName="list-link">
+        <div className="list-link-text">List</div>
+      </NavLink>
+      <NavLink to="/add-item" activeClassName="add-link">
```

Comment:
Great! Now when you refresh, the link stays bold. I think the `activeClassName` could be the same for both links...keeps it simple. It would be a good idea to check the font color and the background color with this tool, to make sure it's accessible: https://webaim.org/resources/contrastchecker/